### ISSUE TITLE
Feature request: Eslint plugin

### ISSUE THREAD

**Issue by Demivan** [fluent-vue/eslint-plugin-fluent-vue#2]
### Discussed in https://github.com/Demivan/fluent-vue/discussions/743

<div type='discussions-op-text'>

<sup>Originally posted by **Demivan** October  4, 2021</sup>
</div>

Check for:
- [ ] Missing translation in the main locale
- [ ] No computed keys
- [ ] Reusing translations keys
- [ ] Invalid fluent syntax

**Comment by Ericlm** [comment]
I think it would be great to report raw text in files, like https://eslint-plugin-vue-i18n.intlify.dev/rules/no-raw-text.html 🙂

**Comment by Ericlm** [comment]
@Demivan would it be okay to make a new scaffold with https://github.com/AlbertHernandez/typescript-library-template?tab=readme-ov-file to have newest deps? I would be willing to do it if needed 🙂

**Comment by Demivan** [comment]
I'm not sure that this project is needed at all. What rules do you think would be useful?

Invalid fluent syntax is now reported by the Vite plugin. Missing translations are better handled by the translation system. `no-raw-text` would be a copy of Intlify one.

**Comment by Ericlm** [comment]
I was thinking about the rules that https://eslint-plugin-vue-i18n.intlify.dev can reports; but I thought it was for vue-i18n only, not for fluent. Maybe am I wrong?

**Comment by Demivan** [comment]
It was designed for vue-i18n. But I see no reason why `no-raw-text`, for example, would not work with fluent-vue.

We are using fluent-vue for a long time now, and I never really thought that we need eslint for it. What vue-i18n rule you find useful for your use case?

**Comment by Ericlm** [comment]
Indeed, `no-raw-text` should still work, but [duplicate keys](https://eslint-plugin-vue-i18n.intlify.dev/rules/no-duplicate-keys-in-locale.html) especially with multiple files, is quite important. And in the same way, [missing keys](https://eslint-plugin-vue-i18n.intlify.dev/rules/no-missing-keys.html), or [dynamic ones](https://eslint-plugin-vue-i18n.intlify.dev/rules/no-dynamic-keys.html).

But I agree on the fact that this would be nice but not essential.

**Comment by Demivan** [comment]
I'm fine with merging PRs if you want to implement the functionality :)
